modules/fbs-core/api/src/main/resources/migrations/05_semester_field_added.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+
+CREATE TABLE IF NOT EXISTS `fbs`.`semester` (
+    `semester_id` INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(100) NOT NULL DEFAULT '',
+    PRIMARY KEY (`semester_id`))
+    ENGINE = InnoDB
+    DEFAULT CHARACTER SET = utf8mb4
+    COLLATE = utf8mb4_0900_ai_ci;
+
+alter table course
+    add `semester_id` INT NOT NULL,
```

Comment:
```suggestion
    add `semester_id` INT,
```
I would, if the course has no semester, set this value to NULL and not to `0`. Furthermore, `0` would not work either, because sql would complain that there is no entry with the key `0` in the table `semester`.